build/openwisp_controller/Dockerfile

```diff
@@ -1,21 +1,15 @@
-FROM python:3.7-alpine AS BASE
+FROM atb00ker/ready-to-run:openwisp-base
```

Comment:
what is `atb00ker/ready-to-run`?

I would like this to be in the openwisp organization since the beginning. I think @hispanico created one a while ago.

Reply:
I created this: https://github.com/openwisp/docker-openwisp/issues/29
We can discuss it there and solve it next week.